src/jdk.jfr/share/conf/jfr/default.jfc

```diff
@@ -1023,6 +1048,8 @@
 
       <flag name="class-loading-enabled" label="Class Loading">false</flag>
 
+      <flag name="container-events-enabled" label="Container Events">true</flag>
```

Comment:
I’d love to know y we need to call at all?

Reply:
Ok. So what would be a good rule-of-thumb for when one should introduce a flag?

Reply:
I think we want to limit the number flags/options There are already too many, preferably we would have none, but some of the ones we have today, like gc and exception, are necessary, because the impact of have them on by default would be catastrophic (long stop the world pauses and possibly million of events per second).

Reply:
Thanks!
Removed the flag.